charts/stable/rtmpserver/docs/installation_notes.md

```diff
@@ -0,0 +1,43 @@
+# Installation Notes
+
+The app has no configuration. The following is just a basic guide on how to stream and view the live stream.
+
+## Stream
+
+- This is the URL to use to stream `rtmp://<server_ip>:1935/live/<stream_key>`
+  - replace <server_ip> and <stream_key> respectively.
+  - change the port if its not default.
+
+## View
+
+- VLC
+
+  - Go to Media > Open Network Stream.
+    /code/charts/truecharts/charts/charts/rtmpserver - `rtmp://<server ip>:1935/live/<stream-key>`
+
+- Web
+  - ~~To play RTMP content: `http://<server_ip>:10610/players/rtmp.html`~~ (requires Flash, this tech is dead!)
+  - To play HLS content: `http://<server_ip>:10610/players/hls.html`
+  - To play HLS content using hls.js library: `http://<server_ip>:10610/players/hls_hlsjs.html`
+  - To play DASH content: `http://<server_ip>:10610/players/dash.html`
+  - To play RTMP and HLS contents on the same page: `http://<server_ip>:10610/players/rtmp_hls.html`
+
+:::note
+
+The default basic builtin stream sites use a default key, "test".
+
+:::
+
+## Customizaton
+
+- If you want to mount a modified version of a ngix.conf you can do so by manually mounting `/etc/nginx/` as PVC or hostpath and then add the custom files after. Take a look at upstream's [configs](https://github.com/TareqAlqutami/rtmp-hls-server/tree/master/conf).
```

Comment:
```suggestion
- If you want to mount a modified version of a nginx.conf you can do so by manually mounting `/etc/nginx/` as PVC or hostpath and then add the custom files after. Take a look at upstream's [configs](https://github.com/TareqAlqutami/rtmp-hls-server/tree/master/conf).
```

Reply:
Also `listen` ports must be plumbed with the targetPorts

Reply:
We could technically have those confs and via dropdown select which one to use.
And also expose couple of settings